### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by b-bodnar**
# PR Details

Added two SQL files with DDL and DML scripts.

### CODE REVIEW

File: task_1_create_db_and_tables.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
+-- create DB
+CREATE SCHEMA `company_employee` DEFAULT CHARACTER SET utf8;
+
+USE company_employee;
+
+-- create table Region
+CREATE TABLE `region`
+(
+    `id`   bigint      NOT NULL AUTO_INCREMENT,
```
Comment (Andriireva) [inline task_1_create_db_and_tables.sql:9]: type should by bigserrial and without AUTO_INCREMENT

File: task_1_create_db_and_tables.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    `name` varchar(45) NOT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `name_UNIQUE` (`name`)
+)
+
+-- create table Company
+CREATE TABLE `company`
+(
+    `id`            BIGINT      NOT NULL AUTO_INCREMENT,
```
Comment (Andriireva) [inline task_1_create_db_and_tables.sql:18]: same as above

File: task_1_create_db_and_tables.sql [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+);
+
+-- create table Employee
+CREATE TABLE `employee`
+(
+    `id`                         BIGINT NOT NULL AUTO_INCREMENT,
+    `first_name`                 VARCHAR(45) NULL,
+    `last_name`                  VARCHAR(45) NULL,
+    `position`                   VARCHAR(45) NULL,
+    `married`                    TINYINT NULL,
```
Comment (Andriireva) [inline task_1_create_db_and_tables.sql:45]: there is type boolean in PostgresSQL

File: task_1_create_db_and_tables.sql [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        FOREIGN KEY (`region`)
+            REFERENCES `company_employee`.`region` (`id`)
+            ON DELETE NO ACTION
+            ON UPDATE NO ACTION
+);
+
+-- create table Employee
+CREATE TABLE `employee`
+(
+    `id`                         BIGINT NOT NULL AUTO_INCREMENT,
```
Comment (Andriireva) [inline task_1_create_db_and_tables.sql:41]: save as above

**Review by Andriireva — CHANGES_REQUESTED** [review]

File: task_1_create_db_and_tables.sql (hunk 1 above)
Comment (b-bodnar) [reply task_1_create_db_and_tables.sql]: Fixed

File: task_1_create_db_and_tables.sql (hunk 2 above)
Comment (b-bodnar) [reply task_1_create_db_and_tables.sql]: Fixed

File: task_1_create_db_and_tables.sql (hunk 3 above)
Comment (b-bodnar) [reply task_1_create_db_and_tables.sql]: Fixed

File: task_1_create_db_and_tables.sql (hunk 4 above)
Comment (b-bodnar) [reply task_1_create_db_and_tables.sql]: Fixed

**Review by Andriireva — APPROVED** [review]

**Pull request merged**